src/main/java/net/mcreator/ui/workspace/WorkspacePanelLocalizations.java

```diff
@@ -417,7 +417,7 @@ private void newLocalizationDialog() {
 
 	@Override public void refilterElements() {
 		for (TableRowSorter<TableModel> sorter : sorters)
-			sorter.setRowFilter(RowFilter.regexFilter(((WorkspacePanel) workspacePanel).search.getText()));
+			sorter.setRowFilter(RowFilter.regexFilter(workspacePanel.search.getText()));

```

Comment:
This and others related should also use convenience method

Reply:
:heavy_check_mark: Fixed